ansible/roles/content-snapshot-indexer/defaults/main.yml

```diff
@@ -9,7 +9,8 @@ redis:
   host: "{{metadata_redis_host}}" # Redis DP IP 
   port: "6379"
   user:
-    index: "12" # For temporary will change once verification is done
```

Comment:
don't change the index value. if you run the migration script to populate the data from cassandra to redis then it will index to 4 instead of 12.

Reply:
Done